index.html

```diff
@@ -63,8 +63,447 @@ <h2 class="who-am-i">I’m a software developer</h2>
+        </section>         
+        <div class="collapse-button">
+            <img src="asset/images/Disabled.png" alt="collapse project-button">
+        </div>          
+
+        <h2 class="works-heading"> MY RECENT WORKS</h2>            
+
+        <section class="place-holder page-1">
+            
+            <div class="post-stories">
+
+                
+                <h2 class="multi-post">
+                        Multi-Post Stories  
+                </h2> 
+                <div class="gain">
+                    Gain+Glory
+                </div>             
+
+                <ul class="lang">
+                    <li >
+                        <button class="button" type="button">
+                            Ruby on rails
+                        </button>
+                        
+                    </li>
+    
+                    <li class="css">
+                        <button class="button" type="button"> 
+                            css
+                        </button>
+                        
+                    </li>
+    
+                    <li class="java">
+                        <button class="button" type="button">
+                            JavScript
+                        </button>
+                        
+                    </li>
+    
+                    <li class="html">
+                        <button class="button" type="button">
+                            html
+                        </button>
+                       
+                    </li>
+                </ul>
+
+                
+
+                <button class="button project-button" type="button">
+                    See Project
+                </button>
+
+            </div>
+        </section>
+
         
+        <section class="place-holder page-2">
+            
+            <div class="post-stories">
+                <h2 class="multi-post">
+                    Multi-Post Stories 
+                </h2>
+                <div class="gain">
+                    Gain+Glory
+                </div>
+
+                <ul class="lang">
+                    <li class="ruby">
+                        <button class="button" type="button">
+                            Ruby on rails
+                        </button>
+                        
+                    </li>
+    
+                    <li class="css">
+                        <button class="button" type="button">
+                            css
+                        </button>
+                        
+                    </li>
+    
+                    <li class="java">
+                        <button class="button" type="button">
+                            JavScript
+                        </button>
+                        
+                    </li>
+    
+                    <li class="html">
+                        <button class="button" type="button">
+                            html
+                        </button>
+                       
```

Comment:
- [ ] I would recommend removing the `button` tag here. Since this is just a text and not some button that will work anything.

Reply:
Button tag has been removed.